requirements.txt

```diff
@@ -1,3 +1,4 @@
 pyarrow>=12.0.1
```

Comment:
Should we update any of these other dependencies while we are at it?

Reply:
Will do a refresh of requirements on the `version=1.0`